diff --git a/dist/index.d.ts b/dist/index.d.ts
index 908b76e49840541ecf46cf57c1740004aa7cf2df..f03c1b05337442350c36c98dd2ece5472026f4d0 100644
--- a/dist/index.d.ts
+++ b/dist/index.d.ts
@@ -1,11 +1,11 @@
-import type { CompileOptions, Warning } from "svelte/types/compiler/interfaces";
+import type { CompileOptions_1, Warning_1 } from "svelte/types/compiler/interfaces";
 import type { PreprocessorGroup } from "svelte/types/compiler/preprocess";
 import type { Plugin } from "esbuild";
 interface esbuildSvelteOptions {
     /**
      * Svelte compiler options
      */
-    compilerOptions?: CompileOptions;
+    compilerOptions?: CompileOptions_1;
     /**
      * The preprocessor(s) to run the Svelte code through before compiling
      */
@@ -31,7 +31,7 @@ interface esbuildSvelteOptions {
      * A function to filter out warnings
      * Defaults to a constant function that returns `true`
      */
-    filterWarnings?: (warning: Warning) => boolean;
+    filterWarnings?: (warning: Warning_1) => boolean;
 }
 export default function sveltePlugin(options?: esbuildSvelteOptions): Plugin;
 export {};
diff --git a/dist/index.js b/dist/index.js
index 44bbb291381f23357ead4285df0103d0abf3e27b..f1d29d0c74387cc8b16f69e96444120f5e9b89d1 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -158,7 +158,7 @@ function sveltePlugin(options) {
           }
           let contents = js.code + `
 //# sourceMappingURL=` + toUrl(js.map.toString());
-          if ((compilerOptions.css === false || compilerOptions.css === "external") && css.code) {
+          if ((compilerOptions.css === false || compilerOptions.css === "external") && css) {
             let cssPath = args.path.replace(".svelte", ".esbuild-svelte-fake-css").replace(/\\/g, "/");
             cssCode.set(
               cssPath,
diff --git a/dist/index.mjs b/dist/index.mjs
index 5fd4f090849e55934ab7ee15e9500a51cfafb276..35d29ee73b9545116cf3f09ecfffd1c9a9047856 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -158,7 +158,7 @@ function sveltePlugin(options) {
           }
           let contents = js.code + `
 //# sourceMappingURL=` + toUrl(js.map.toString());
-          if ((compilerOptions.css === false || compilerOptions.css === "external") && css.code) {
+          if ((compilerOptions.css === false || compilerOptions.css === "external") && css) {
             let cssPath = args.path.replace(".svelte", ".esbuild-svelte-fake-css").replace(/\\/g, "/");
             cssCode.set(
               cssPath,
